presto-spi/src/test/java/com/facebook/presto/spi/type/TestUnscaledDecimal128Arithmetic.java

```diff
@@ -679,4 +753,38 @@ private static Slice negateConstructive(Slice slice)
         negate(copy);
         return copy;
     }
+
+    private static int[] toInt8Array(BigInteger value)
+    {
+        byte[] bigIntegerBytes = value.toByteArray();
+        reverse(bigIntegerBytes);
```

Comment:
Instead of making reverse public, use `Collections.reverse(Bytes.asList(data))`.